modules/install/pages/pre-install.adoc

```diff
@@ -35,3 +35,22 @@ However, you must adhere to the Minimum Specifications for production._
 
 **_The Recommended Specifications don't take into account your intended workload.
 You should follow the xref:sizing-general.adoc[sizing guidelines] when determining system specifications for your Couchbase Server deployment._
+
+[#clock-source-linux]
+Clock Source on Linux:: The Query service uses the OS monotonic clock for profiling and network timeout purposes. 
++
+The Linux kernel uses the _Clock Source_ to obtain the current clock value and this information is stored in `/sys/devices/system/clocksource/clocksource0/current_clocksource`. There are several clock sources (TSC, XEN, and others), which are used depending on the hardware clock capabilities, and the OS installation. The XEN source, which is seen to be the default on AWS setups, can use up to 25% of all available CPU time to obtain the current timestamp. The TSC clock source, on the other hand, incurs very little CPU cost. We recommend changing the clock source to TSC if it is set to anything else.
++
+Check the clock source on your Linux OS using the following command:
+[source, bash]
+----
+cat /sys/devices/system/clocksource/clocksource0/current_clocksource
+----
++
+Change the clock source using the following commands:
+[source,bash]
+----
+echo tsc > /sys/devices/system/clocksource/clocksource0/current_clocksource
+
+cat /sys/devices/system/clocksource/clocksource0/current_clocksource tsc
```

Comment:
'tsc' is the output of the cat command, so it should be on the next line.

Even better, you could say

Change the clock source using the following command:

echo tsc > /sys/devices/system/clocksource/clocksource0/current_clocksource

To verify the current setting of the clock source, use:

 cat /sys/devices/system/clocksource/clocksource0/current_clocksource

The output should read 'tsc'

Reply:
Updated.